zerver/lib/realm_logo.py

```diff
@@ -23,7 +24,9 @@ def get_realm_logo_url(realm: Realm, night: bool) -> str:
         else:
             logo_version = realm.logo_version
         return upload_backend.get_realm_logo_url(realm.id, logo_version, night)
-    return settings.DEFAULT_LOGO_URI + "?version=0"
+    if settings.DEFAULT_LOGO_URI is not None:
+        return settings.DEFAULT_LOGO_URI + "?version=0"
+    return staticfiles_storage.url("images/logo/zulip-org-logo.svg") + "?version=0"
```

Comment:
Ideally Django would hash this URL and we wouldn’t need `?version=0` (which…maybe should have been bumped in 8dc3e086c9cf1f7347f0b4fe0360dfd631cf1ab7). However, we are currently preventing Django from hashing it.

https://github.com/zulip/zulip/blob/8dc3e086c9cf1f7347f0b4fe0360dfd631cf1ab7/zerver/lib/storage.py#L39-L48

This could be a later followup task.

Reply:
I assume you mean some other commit than 8dc3e086c9cf1f7347f0b4fe0360dfd631cf1ab7 -- presumably 9613c665600a8bae0159e93bd4b30c9741c554e9?

I noticed that none of the `staticfiles_storage.url` were hashed; I ran into trouble flipping this block off (`Post-processing 'generated/emoji/twitter-sprite.css' failed` and `The file 'generated/emoji/~emoji-datasource-twitter/img/twitter/sheets-256/64.png' could not be found`), so I deferred doing so to a follow-up.

Reply:
Er, I meant c2f9db4602a1b88285036313006332d7d0b0782e actually. The point of `version` was to be bumped when the image changed, so we should have bumped it when the image changed.